tests/lib/rules/no-inline-comments.js

```diff
@@ -40,6 +40,12 @@ ruleTester.run("no-inline-comments", rule, {
         "var a = 1; // eslint-disable-line no-debugger",
         "var a = 1; /* eslint-disable-line no-debugger */",
 
+        // https://github.com/eslint/eslint/issues/14575
+        "var i;//eslint no-debugger:2",
+        "var i;//eslint-disable no-debugger",
+        "var i;//eslint-enable no-debugger",
+        "var i;//global foo",
```

Comment:
Same, can we mix in a couple comments with spaces after `//`?

Reply:
addressed in a2b3aaa46146f794a006b8f4014502d61ade9b85